### PR TITLE
XCM Utils Precompile PR v2: electric boogaloo

### PR DESCRIPTION
**Pull request by jboetticher**
### Description

Added details about the XCM Utils precompile into the remove-evm-calls page. 

Claimed that multilocationToAddress would get the multilocation-derived account of a multilocation, but I'm not sure about that claim. [The rust code](https://github.com/PureStake/moonbeam/blob/94d71502ba31372c912eb8e9117536d4f1c09320/precompiles/xcm-utils/src/lib.rs#L68
)  didn't give me too much more confidence. Hopefully @albertov19 can confirm

### Checklist

- [X] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Review by eshaben — CHANGES_REQUESTED** [review]
Sorry for the delay in reviewing this! 

It might be nice to show an example of how the utility functions can be used 🤷‍♀️

**Review by eshaben — APPROVED** [review]
LGTM... @albertov19 do you want to review before I merge?

**Comment by jboetticher** [comment]
@eshaben I talked to Alberto and he said that I should put it in its own page, so expect a new commit soon. Sorry I didn't let you know earlier

**Comment by jboetticher** [comment]
It was ultimately concluded by Gorka that the XCM-Utils precompile isn't extremely useful, so I'm not too stressed about the content of the page. Still good to have something on it, I guess

**Review by eshaben — COMMENTED** [review]
Not sure how to feel about this as a standalone page. The introduction is a bit short. I'm not sure... I'm going to defer to @albertov19 to review this 🤷‍♀️

**Comment by jboetticher** [comment]
Index image seems to be left over from a previous change I made while editing. Removed it @albertov19 if you want to approve

File: builders/pallets-precompiles/precompiles/xcm-utils.md
Comment (wuxinxin2000) [inline builders/pallets-precompiles/precompiles/xcm-utils.md:45]: The `Multilocation` struct in the XCM-utils precompile is built the same as [the XCM-transactor precompile's `Multilocation`](/builders/xcm/xcm-transactor#building-the-precompile-multilocation){target=_blank}.

Comment (eshaben) [reply builders/pallets-precompiles/precompiles/xcm-utils.md:45]: @wuxinxin2000 thanks for suggesting this fix. Since this PR has already been closed, would you mind opening a new one with this change please? You can request my review on it and I'll merge it in 🙂

**Pull request merged**
